bmt/bmt.go

```diff
@@ -688,3 +795,11 @@ func calculateDepthFor(n int) (d int) {
 	}
 	return d + 1
 }
+
+// LengthToSpan creates a binary data span size representation
+// It is required for calculating the BMT hash
+func LengthToSpan(length int) []byte {
```

Comment:
This function can be unexported.

Reply:
It will be used by other components: https://github.com/ethersphere/swarm/pull/2022